nilearn/glm/_base.py

```diff
@@ -1,10 +1,10 @@
 from nibabel.onetime import auto_attr
-from sklearn.base import BaseEstimator, TransformerMixin
+from sklearn.base import BaseEstimator
 
 from nilearn._utils import CacheMixin
 
 
-class BaseGLM(TransformerMixin, CacheMixin, BaseEstimator):
```

Comment:
as far I can tell the GLM estimators do not need the transformers functionalities